manifest.json

```diff
@@ -518,6 +518,17 @@
          ],
          "type" : "color",
          "version" : "0.1"
+      },
+      {
+         "id" : "focus-theme",
```

Comment:
Might be a good idea to just call it `focus`. Same for the file name.
What do you think?

Reply:
i like it i prefer focus i was going to call it focus initially but i  thought it was more professional ™